packages/application-shell/src/components/version-tracker/version-tracker.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import flowRight from 'lodash/flowRight';
+import { connect } from 'react-redux';
+import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import applicationShellVersion from '../../version';
+import pkg from '../../../package.json';
+import * as actions from './actions';
+
+const createVersionMetric = ({ applicationName }) => ({
+  metricName: 'versions',
+  metricLabels: {
+    application: applicationName || 'unknown',
+    package_name: pkg.name,
```

Comment:
This may look a bit odd but the idea was that we could also send more versions of more packages back given they often are not updated.